JSDemos/Demos/TreeList/MultipleRowSelection/AngularJS/index.js

```diff
@@ -34,7 +35,17 @@ DemoApp.controller('DemoController', function DemoController($scope) {
                 width: 120
             }
         ],
-        expandedRowKeys: [1, 2, 10]
+        expandedRowKeys: [1, 2, 10],
+        onSelectionChanged: function(e) {
+            var data = e.component.getSelectedRowsData("all");
+    
+            if(data.length > 0)
+                $scope.selectedItemsText = $.map(data, function(value) {
+                    return value.Full_Name;
+                }).join(", ");
+            else 
+                $scope.selectedItemsText = "Nobody has been selected";
+        }
```

Comment:
I think it is better to use the same code style in all approaches:
```suggestion
        onSelectionChanged: function(e) {
            const data = e.component.getSelectedRowsData("all");
    
            if(data.length > 0)
                $scope.selectedItemsText = data.map(value => value.Full_Name).join(", ");
            else 
                $scope.selectedItemsText = "Nobody has been selected";
        }
```